src/resources/container.js

```diff
@@ -9,15 +9,35 @@ import { GlbParser } from './parser/glb-parser.js';
 /**
  * @class
  * @name pc.ContainerResource
- * @classdesc Container for a list of animations, textures, materials and a model.
+ * @classdesc Container for a list of animations, textures, materials, models, nodes, scenes, default scene
+ * cameras and lights. Entities in scene hierarchies will have model, camera and light components attached to them.
+ * Animation components have to be added manually (using nodeAnimations) as either pc.AnimComponent or pc.AnimationComponent.
  * @param {object} data - The loaded GLB data.
+ * @property {pc.Entity|null} scene Root entity of the default GLB scene.
+ * @property {pc.Entity[]} scenes Root entities of scenes indexed by GLB scenes.
+ * @property {pc.CameraComponent[]} cameras Instanced camera components, does not match index of GLB cameras.
+ * @property {pc.LightComponent[]} lights Instanced light components, does not match index of GLB lights.
+ * @property {pc.Entity[]} nodes Entities indexed by GLB nodes.
+ * @property {pc.Asset[]} materials Material assets indexed by GLB materials.
+ * @property {pc.Asset[]} textures Texture assets indexed by GLB textures.
+ * @property {pc.Asset[]} animations Animation assets indexed by GLB animations.
+ * @property {number[][]} nodeAnimations Animation asset indices indexed by GLB nodes.
+ * @property {pc.Asset[]} models Model assets indexed by GLB meshes.
+ * @property {pc.AssetRegistry} registry The asset registry.
  */
 function ContainerResource(data) {
     this.data = data;
-    this.model = null;
+    this.scene = null;
+    this.scenes = [];
+    this.cameras = [];
+    this.lights = [];
+    this.nodes = [];
```

Comment:
What do think of using the term `entites` instead of `nodes`? To more accurate convey the type.

Reply:
I considered this when I started implementing these features but opted for `nodes` in the end for two reasons:
* `entities` would be equally applicable to the property `scenes` since that is also a list of entities.
* The `nodes` property is mainly used to apply extensions to certain nodes, hence it is important to convey that it uses the same indexing as the node list in the glTF. In my opinion this is more important than to convey the type.